src/utilities/attachments-helper.ts

```diff
@@ -9,7 +9,9 @@ interface AttachmentHelperOptions {
 };
 
 interface RequestPresignedURLResponse {
-    upload_url: string;
+    // TODO: URL type?
+    uploadURL: string;
+    photoFilename: string;
```

Comment:
has nothing to do with this code but I don't like the name photoFilename unless it will always be a photo
I thought the lambda generated pdfs?

Reply:
We should get the lambda code into a repository. This was due to reverting back to an older Node.JS script, rather than the newer Python script that you had originally integrated with.